src/Money.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare(strict_types=1);
+
+namespace Landingi\Core;
+
+class Money
+{
+    private float $value;
```

Comment:
`float` to dobry pomysł? :D Ja np mam 3 znaki po przecinku w ECO :)

Reply:
a to float w php ma tylko 2 znaki po przecinku? To jest określone w https://en.wikipedia.org/wiki/IEEE_754

Reply:
`float` jako "money" to zawsze zły pomysł

Reply:
Nie, ale z definicji powinniśmy mieć obsługę 4 znaków po przecinku, `float` w PHP ma tendencję do kłamania, można go ograniczać poprzez `php.ini` itp. W sumie to pewnie w większości aplikacji mamy `float` w cenie.

Reply:
Yep, generalnie to się trzyma całości jako jeden int i "grosze" (u nas generalnie chyba się to nazywa `cents`) jako osobny int. Lepiej nie polegać na operacjach zmiennoprzecinkowych bo różne cuda potem wychodzą

Reply:
W takim razie jak sie powinno zachować getValue? zwrócić stringa, czy jaka zwrotke?